.golangci.yml

```diff
@@ -4,7 +4,6 @@ linters-settings:
 
 linters:
   enable:
-    - goconst
```

Comment:
not possible with `// nolint:goconst` ?

Reply:
I don't believe this check is relevant, to be honest.